### PR TITLE
FIX: Don't publish PM archive events to acting user.

### PR DESCRIPTION
**Pull request by tgxworld**
When a user archives a personal message, they are redirected back to the
inbox and will refresh the list of the topics for the given filter.
Publishing an event to the user results in an incorrect incoming message
because the list of topics has already been refreshed.

This does mean that if a user has two tabs opened, the non-active tab
will not receive the incoming message but at this point we do not think
the technical trade-offs are worth it to support this feature. We
basically have to somehow exclude a client from an incoming message
which is not easy to do.

Follow-up to fc1fd1b41689694b3916dc4e10605eb9b8bb89b7

### CODE REVIEW

**Review by eviltrout — APPROVED** [review]

**Comment by tgxworld** [comment]
Thank you for the review @eviltrout :hugs:
